diesel/src/pg/expression/expression_methods.rs

```diff
@@ -683,3 +683,592 @@ where
     T::SqlType: RangeOrNullableRange,
 {
 }
+
+/// PostgreSQL specific methods present between CIDR/INET expressions
+pub trait PgNetExpressionMethods: Expression + Sized {
+    /**
```

Comment:
Any reason why this does not use the standard rust-doc comments (`///`)? 
(The same applies for all methods below)

Reply:
No specific reason, just a personal habit of mine of using `/* ... */` style for multiline comments regardless of the language. Will change should you prefer `///` (`cargo fmt` does not change it which is surprising).

Reply:
Changed in 486373b22